config/rplidar_a3.yaml

```diff
@@ -1,6 +1,6 @@
 rplidar:
   ros__parameters:
-    serial_port: /dev/rplidar
+    serial_port: /dev/ttyUSB0
```

Comment:
Should this be rplidar? I am not sure if the usage of indoor/outdoor configuration is enabled yet.

Reply:
I removed unnecessary configs and left just only one that is really used.